index.html

```diff
@@ -0,0 +1,46 @@
+<DOCTYPE html>
+  <html>
+    <head>
+      <title>My web page</title>
+    </head>
+    <body>
+      <h1>Clayton
+        
+      </h1>
+      <h2>
+         computer
+      </h2>
+      <h3>
+        gamer all day everyday
+      </h3>
+      <hr>
+      <p>
+        testing testing. 
+        <section id="testing">
+      </section>
+      </p>sentence
+      <h3>
+        accomplishments
+      </h3>
+      <ul>
+        <li><strong>hello world</li>
```

Comment:
It looks like you're missing a `</strong>` here - that's why the entire rest of the list is bold. HTML is funny that way. :donut: